raspberrypi-node.js

```diff
@@ -65,7 +65,7 @@ function initTasks(gulp, options) {
 
     all.uploadFilesViaScp(filesLocal, filesRemote, function () {
       console.log("- Installing NPM packages on the device. It might take several minutes.");
-      all.sshExecCmd('cd ' + targetFolder + ' && chmod 600 *.pem && npm install', { verbose: args.verbose }, cb);
+      all.sshExecCmd('cd ' + targetFolder + ' && ls *.pem &>/dev/null && chmod -f 600 *.pem || : && npm install', { verbose: args.verbose }, cb);
     });
```

Comment:
Kind of confusing to me. I searched a bit and have an alternative implementation pasted as below.
```bash
cd app && file=(*.pem) && if [ -e "$file" ]; then chmod -f 600 *.pem; fi && npm install
```